libs/react-helpers/src/lib/links/links.ts

```diff
@@ -0,0 +1,27 @@
+/** createDocsLinks returns external documentation links that are specific to an environment
+ * @param {string} docsUrl - the documentation URL for the environment ex. https://docs.vega.xyz/testnet or https://docs.vega.xyz (NX_VEGA_DOCS_URL)
+ */
+
+export const createDocsLinks = (docsUrl: string) => ({
+  AUCTION_TYPE_OPENING: `${docsUrl}/concepts/trading-on-vega/trading-modes#auction-type-opening`,
+  AUCTION_TYPE_LIQUIDITY_MONITORING: `${docsUrl}/concepts/trading-on-vega/trading-modes#auction-type-liquidity-monitoring`,
+  AUCTION_TYPE_PRICE_MONITORING: `${docsUrl}/concepts/trading-on-vega/trading-modes#auction-type-price-monitoring`,
+  AUCTION_TYPE_CLOSING: `${docsUrl}/concepts/trading-on-vega/trading-modes#auction-type-closing`,
+});
+
+export const ExternalLinks = {
+  FEEDBACK: 'https://github.com/vegaprotocol/feedback/discussions',
+  GITHUB: 'https://github.com/vegaprotocol/token-frontend',
+  DISCORD: 'https://vega.xyz/discord',
+  STAKING_GUIDE:
```

Comment:
shouldn't this (and anything else with an environment hard coded) be in the generate docsLink function?

Reply:
updated in last commit.